cluster/gce/config-test.sh

```diff
@@ -377,3 +377,6 @@ ENABLE_POD_PRIORITY="${ENABLE_POD_PRIORITY:-}"
 if [[ "${ENABLE_POD_PRIORITY}" == "true" ]]; then
     FEATURE_GATES="${FEATURE_GATES},PodPriority=true"
 fi
+
+# Optional: enable certificate rotation of the kubelet certificates.
+ROTATE_CERTIFICATES="${ROTATE_CERTIFICATES:-true}"
```

Comment:
I'm not a fan of having vastly different default config in -default and -test. We want to test things that we ship, not some other config. Please disable it by default in test config and create a special test suite to test certificate rotation.

Reply:
Done.